src/reflect.js

```diff
@@ -63,3 +63,8 @@ if (typeof Reflect.construct !== 'function') {
     return new (bind.apply(Target, a));
   };
 }
+
+if (typeof Reflect.ownKeys !== 'function')
+    Reflect.ownKeys = function (o) {
+    return (Object.getOwnPropertyNames(o).concat(Object.getOwnPropertySymbols(o))));
```

Comment:
Sorry all, was afk for a while. Object.getOwnPropertySymbols didn't need to be pollyfilled on IE11 (which is where I have tested this), but I imagine it could be needed elsewhere. I'll add it and rebase the PR now.